docs/getting-started.md

```diff
@@ -0,0 +1,542 @@
+ # Getting Started with Tekton Pipelines
+
+This guide will get you started with Tekton Pipelines with a simple `Hello World` tutorial.
```

Comment:
These two sentences are rather redundant. I would say:

"This guide helps you get started with Tekton Pipelines by walking you through a simple `Hello World` tutorial. The tutorial shows you how to:"